### PR TITLE
[Bug](agg-state) fix core dump on not nullable argument for aggstate's nested argument

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes

1. fix core dump on not nullable argument for aggstate's nested argument
2. add limit for wrong function signature on aggstate
3. fix insert into values to table have mv failed.
4. add '()' to ArithmeticExpr's tosql, Make old planner behave the same as nereids.
## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by zhangstar333 — DISMISSED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/combinator/MergeCombinator.java [hunk 1]
```diff
@@ -47,13 +46,12 @@ public MergeCombinator(List<Expression> arguments, AggregateFunction nested) {
         super(nested.getName() + AggStateFunctionBuilder.MERGE_SUFFIX, arguments);
 
         this.nested = Objects.requireNonNull(nested, "nested can not be null");
-        inputType = new AggStateType(nested.getName(), nested.getArgumentsTypes(),
-                nested.getArguments().stream().map(Expression::nullable).collect(Collectors.toList()));
+        inputType = (AggStateType) arguments.get(0).getDataType();
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/combinator/MergeCombinator.java:49]: please check if StateCombinator and UnionCombinator should done the same thing?

**Review by morrySnow — DISMISSED** [review]

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Review by zhangstar333 — APPROVED** [review]

**Pull request merged**